content/content-delivery/v1/core-resources/collaborators/collaborators.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Collaborators
```

Comment:
This is not working for me, I guess it's probably something related to the MAPI, not to the CDN API

Reply:
I send this request to the CDN API (as this is the folder of the CDN API) and the endpoint is not working. Also, it contains the story id, but I guess it should contain just the space id.